contacts added

Plov3 opened the pull request.

Plov3 committed:
contacts added

Makoto-kino14 commented on src/partials/contacts.html:
```diff
@@ -0,0 +1,86 @@
+                <p class="contacts__list-description">Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
+            </div>
+            <div class="contacts__list-wrap">
+                <h4 class="contacts__list-header">Monday - Friday</h4>
```
<!--// между Saturday-Sunday и часами работы нет margin-bottom. Снять класс-->

Makoto-kino14 commented on src/sass/components/_modal-btn.scss:
```diff
@@ -1,31 +1,31 @@
 .modal-btn {
```
Удали пож-та этот коммит, не вижу изменений. Но надо доработать кнопки (добавить padding; псевдоэлемент after - стрелочка)

Plov3 replied:
да, все один в один. не знаю как это убрать. возможно это из-за того, что я изменил там что-то. Потом решил не трогать, чтобы не испортить и отменил действия.

Makoto-kino14 replied:
Удалила файл в ветке feature/contacts

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+.contacts__list {
```
// секции contacts надо дать background-color:#FFF5F6;

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+    display: flex;
+  }
+}
+.contacts__list-item {
```
// нет стилей границ

Plov3 replied:
не понял каких границ нет(

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 40px;
```
// padding для верха и низа должен быть равен правому и левому (десктопная версия)

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+  text-align: center;
+  letter-spacing: 0.04em;
+  color: #ffffff;
+  padding: 3px 24px;
```
// padding вверху и внизу задавать не надо

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+    font-size: 16px;
+  }
+}
+.contacts__list-description {
```
// почему то на макете отображен перенос другого слова

Plov3 replied:
на макете зачем-то ширина поля для текста ограничена. исправил

Makoto-kino14 replied:
Расул, проверь третью ли-шку в ul contacts__list. При ширине от 1200 до 1279 px она шире других

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+    font-size: 16px;
+  }
+}
+.contacts__working-hours {
```
// убери margin-bottom с last-child (вторые часы работы на макете)

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+  display: block;
+  font-weight: 500;
+  line-height: 1.86;
+  color: #ffa5ba;
```
// вставить цвет - переменную

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+    margin-bottom: 12px;
+  }
+}
+.contacts__list-wrap {
```
// padding и margin снизу для десктопа и планшета - 32px

Makoto-kino14 commented on src/sass/layouts/_contacts.scss:
```diff
@@ -0,0 +1,158 @@
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #d41443;
```
// вставить цвет - переменную

Plov3 committed:
bugs_fixed

Plov3 committed:
Update contacts.html

поправил по семантике проекта

Makoto-kino14 committed:
Delete _modal-btn.scss

Расул, пытаюсь удалить у тебя этот паршл : ) после того как апрувнешь, сделай pull из ветки main

Makoto-kino14 commented on src/partials/contacts.html:
```diff
@@ -0,0 +1,86 @@
+            </div>
+        </li>
+    </ul>
+    <ul class="contacts__btn list">
```
Кнопки отображаются только при ширине 1200 px

Makoto-kino14 reviewed (changes requested):
![image](https://user-images.githubusercontent.com/103353487/178227554-3012eb8e-056a-4afa-bf1e-cc051b3b1238.png)
 Расул, что то с 3ей лишкой по ширине (она больше остальных при ширине экрана 1200-1280 px). По кнопкам еще надо доработать
